docs/glossary.rst

```diff
@@ -1155,12 +1155,19 @@ Glossary
    cookiecutter
       A command-line utility that creates projects from :ref:`cookiecutters <cookiecutter:readme>` (project templates), e.g., creating a Python package project from a Python package project template.
 
-      Pyramid cookiecutters include:
+      The unified cookiecutter ``pyramid-cookiecutter-starter`` merged features from the three previous cookiecutters which are now deprecated.
+
+      * `pyramid-cookiecutter-starter <https://github.com/Pylons/pyramid-cookiecutter-starter>`_
+
+
+      .. versionadded:: 1.10
 
       * `pyramid-cookiecutter-alchemy <https://github.com/Pylons/pyramid-cookiecutter-alchemy>`_
       * `pyramid-cookiecutter-starter <https://github.com/Pylons/pyramid-cookiecutter-starter>`_
       * `pyramid-cookiecutter-zodb <https://github.com/Pylons/pyramid-cookiecutter-zodb>`_
 
+      .. deprecated:: 1.10
+
       .. versionadded:: 1.8
 
```

Comment:
Starting with "The unified cookiecutter ... " and ending with ".. versionadded:: 1.8", replace with the following:

```
.. versionadded:: 1.8
    Added cookiecutter support.
 .. versionchanged:: 1.10
    Merged features from ``pyramid-cookiecutter-alchemy`` and ``pyramid-cookiecutter-zodb`` into the single cookiecutter to rule them all, ``pyramid-cookiecutter-starter``.
 .. deprecated:: 1.10
    ``pyramid-cookiecutter-alchemy`` and ``pyramid-cookiecutter-zodb`` are no longer supported.
    Use ``pyramid-cookiecutter-starter`` going forward.
```